.gitignore

```diff
@@ -1,4 +1,6 @@
 .idea
 .composer
+.phpunit.result.cache
 vendor
 composer.lock
+index.php
```

Comment:
You need to remove this entry from this file, push your changes, then remove index file, push changes again and then add this line to git ignore.

Reply:
Done